recipes/ckzg/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ckzg" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ckzg-{{ version }}.tar.gz
+  sha256: 4295acc380f8d42ebea4a4a0a68c424a322bb335a33bad05c72ead8cbb28d118
+  patches:
+    - setup.py.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('clang') }}
+    - {{ stdlib('c') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+  host:
+    - python
+    - pip
+    - libblst
+    - libblst-headers
+  run:
+    - python
+    - libblst
```

Comment:
Appears `libblst` is missing `run_exports`, have raised this discussion in issue: https://github.com/conda-forge/libblst-feedstock/issues/2

Also it appears we don't have `libblst` in the global pinnings. So raised issue: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6623

Would be good to sort these out so we can ensure this is being pinned appropriately

Reply:
﻿Should be resolved once https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6655 is merged.